sdk/core/azure-core/src/main/java/com/azure/core/http/HttpPipelineCallContext.java

```diff
@@ -76,7 +71,7 @@ public Optional<Object> getData(String key) {
      *
      * @return The context associated to the HTTP call.
      */
-    Context getContext() {
+    public Context getContext() {
```

Comment:
Since we are making this public, we should also update the javadoc to mention that the `Context` returned is a snapshot of the context at the time this method was called and updates to the context will not be reflected in the instance returned from this method.